### ISSUE TITLE
Adding more presets?

### ISSUE THREAD

**Issue by iChristGit** [oobabooga/text-generation-webui#364]
Is there a way to add more presets to the webui? Or maybe have a easy way to dl them like the model downloader that is provided?

**Comment by official-elinas** [comment]
There's a folder called `presets` where you can add your own as txt files.

**Comment by Loufe** [comment]
Good idea, had an idea to add a button to UI to save the preset values by a set name. I'll work on it next week and create a PR
